src/components/alerts/alert.jsx

```diff
@@ -25,6 +34,7 @@ const AlertComponent = ({
                 />
             ) : null}
             {message}
+            {content}
```

Comment:
Looks like you want message and content to be separated by a space:
![image](https://user-images.githubusercontent.com/654102/48134284-7ce9b500-e267-11e8-9779-a49dbf9c1464.png)

This is what it looks like on beta:
![image](https://user-images.githubusercontent.com/654102/48134293-84a95980-e267-11e8-9fd4-ac46821df898.png)

Putting these two on the same line and separating with an html entity space like `&nbsp;` will give you the result you want. 

However, this will not work well for translations, where the name of the device may not be at the end of the line. It looks like the message is coming from the VM, so I'm fine with not fixing that here. But can you file an issue on the VM and leave a TODO comment here with the issue number.

Reply:
I don't know if it's an issue with the VM, beyond the need to translate the message itself -- but there's this issue: https://github.com/LLK/scratch-vm/issues/1725

In the meantime, I put in a TODO to handle RTL (I don't know how you reverse the order of multiple items) and added an &nbsp;